libs/cosmos-sdk/x/bank/internal/keeper/keeper_adapter.go

```diff
@@ -0,0 +1,42 @@
+package keeper
```

Comment:
exchain old code have the same keeper_adapter https://github.com/okex/exchain/blob/dev/libs/cosmos-sdk/x/bank/internal/keeperadapter/bankkeeper_adapter.go suggest merge code

Reply:
done